### PR TITLE
Add Flower Labs

### PR DESCRIPTION
**Pull request by jafermarq**
## Add/Update/Remove <CompanyName>

- [x] I have read the [contributing guidelines](https://github.com/poteto/hiring-without-whiteboards/blob/master/CONTRIBUTING.md)
- [x] I agree to the [Code of Conduct](https://github.com/poteto/hiring-without-whiteboards/blob/master/CODE_OF_CONDUCT.md)
- [x] I have followed the [format](https://github.com/poteto/hiring-without-whiteboards/blob/master/CONTRIBUTING.md#format) prescribed in the contributing guidelines
- [ ] (OPTIONAL) In your pull request message, add additional context on the interview process if necessary

### CODE REVIEW

**Comment by jafermarq** [comment]
Hi @poteto , I checked the bors log and found two warnings. I fixed the one about trailing `/`. Not sure what to do about the `Remote in Europe` warning. Do you have any suggestions? I think these two warning caused the build to fail?

File: README.md [hunk 1]
```diff
@@ -332,6 +332,7 @@ Also check out [No Whiteboards](https://www.nowhiteboard.org) to search for jobs
 - [Flatfox](https://flatfox.ch) | Zurich, Switzerland | Informal conversation to check mutual fit, small (4h) take-home assignment, discussion in team
 - [FloorForce](https://jobs.lever.co/adhawk?location=Sarasota%20Florida) | Sarasota, FL / Remote | Intro phone call with HR, second conversation with FloorForce engineer, and final pairing interview with FloorForce engineer over a screenshare
 - [Flowdash](https://flowdash.com/careers) | San Francisco, CA | Technical phone interview, pair programming on-site
+- [Flower Labs](https://flower.dev) | Cambridge, UK / Berlin, Germany / Remote in Europe. Reach out to us by email or on Slack with your CV. You'll receive a written interview which should take 15min to 30min to complete, after which we'll schedule at most 2x technical video interviews with a potential take-home assignment in between. Alternatively, you share the Open-Source code you have written so we can discuss that.
```
Comment (poteto) [inline README.md:335]:
```suggestion
- [Flower Labs](https://flower.dev) | Cambridge, UK; Berlin, Germany; Remote | Reach out to us by email or on Slack with your CV. You'll receive a written interview which should take 15min to 30min to complete, after which we'll schedule at most 2x technical video interviews with a potential take-home assignment in between. Alternatively, you share the Open-Source code you have written so we can discuss that.
```

Comment (poteto) [reply README.md:335]: This should work, our linter isn't super smart. In particular we only allow "Remote" as a location, but if you have a jobs page you could mention there that remote is limited to similar timezones

**Review by poteto — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jafermarq) [reply README.md]: Happy to leave it as "remote"

**Comment by poteto** [comment]
Thanks!

bors r+

**Pull request merged**